mixer/pkg/protobuf/yaml/dynamic/handler.go

```diff
@@ -216,7 +216,11 @@ func (h *Handler) HandleRemoteQuota(ctx context.Context, encodedInstance *adapte
 		return nil, err
 	}
 
-	qRes := result.Quotas[encodedInstance.Name]
+	qRes, found := result.Quotas[encodedInstance.Name]
+	if !found {
+		return nil, errors.Errorf("remoteQuota no quota for:%s available:%v", encodedInstance.Name, result.Quotas)
```

Comment:
let's elaborate this error. Remote server <via handler name> granted quote for xxxx which is not available /... WDYT ?